templates/taxi/car_detail.html

```diff
@@ -3,20 +3,19 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
-  <hr>
-  <ul>
-    {% for driver in car.drivers.all %}
-      <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
-    {% endfor %}
-  </ul>
-{% endblock %}
+  <h1>Drivers
```

Comment:
The `<h1>` tag should be closed before the form starts to maintain proper HTML structure. Consider moving the closing `</h1>` tag to line 6 and using a different tag, such as `<div>` or `<section>`, to wrap the form and its contents.